app/Http/Controllers/PayloadHistoryController.php

```diff
@@ -73,9 +73,15 @@ public function destroy(PayloadHistory $history)
             $this->payloadHistoryRepository->delete($history->id);
 
             return redirect(route('history.index'))
-                    ->with('messageSuccess', 'This payload history successfully deleted');
+                ->with('messageSuccess', [
+                    'status' => 'Delete success',
+                    'message' => 'This payload history successfully deleted'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadHistoryController.php</br>**Line**: 78</br>Each line in an array declaration must end in a comma